react/features/base/session/constants.js

```diff
@@ -0,0 +1,12 @@
+
+export const SESSION_CONFIGURED = Symbol('SESSION_CONFIGURED');
+
+export const SESSION_ENDED = Symbol('SESSION_ENDED');
+
+export const SESSION_FAILED = Symbol('SESSION_FAILED');
+
+export const SESSION_STARTED = Symbol('SESSION_STARTED');
+
+export const SESSION_WILL_END = Symbol('SESSION_WILL_END');
```

Comment:
Generally I don't like these XX_WILL_YY because they end up being unreliable. If you fail you can't say you will fail, so it creates this asymmetric API.